### PR TITLE
Story/dfc 3156 create sitefinity add in to support o net

### PR DESCRIPTION
**Pull request by Gurnaik-Randhawa**
Sitefinity content types to support the ONet framework implementation.

### CODE REVIEW

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/Classifications/Structure/taxonomies.sf [hunk 1]
```diff
@@ -133,11 +133,35 @@
...
   },
+  {
+    "Id": "4697627c-2bb0-4b3f-b455-798d27048deb",
+    "Name": "JobProfileKeywords",
+    "Type": "Telerik.Sitefinity.Taxonomies.Model.FlatTaxonomy",
+    "LastModified": "2018-06-29T08:36:41.407Z",
+    "Title": [
+      {
+        "Key": "",
+        "Value": "Job Profile Keywords"
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/Classifications/Structure/taxonomies.sf]: This should not be there, this is a old taxon.

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/JobProfile.nuspec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
   <metadata>
     <id>JobProfile</id>
-    <version>1.0.27.2</version>
+    <version>1.0.28.1</version>
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/JobProfile.nuspec]: We are in sprint 29, so this should be 1.0.Sprint.Revision = 1.0.29.1

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Classifications/Structure/taxonomies.sf [hunk 3]
```diff
@@ -133,11 +133,35 @@
...
   },
+  {
+    "Id": "4697627c-2bb0-4b3f-b455-798d27048deb",
+    "Name": "JobProfileKeywords",
+    "Type": "Telerik.Sitefinity.Taxonomies.Model.FlatTaxonomy",
+    "LastModified": "2018-07-02T11:12:19.413Z",
+    "Title": [
+      {
+        "Key": "",
+        "Value": "Job Profile Keywords"
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Classifications/Structure/taxonomies.sf]: Same as above JP keywords should not be there

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/Classifications/Structure/taxonomies.sf (hunk 1 above)
Comment (Gurnaik-Randhawa) [reply DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/Classifications/Structure/taxonomies.sf]: Removed

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/JobProfile.nuspec (hunk 2 above)
Comment (Gurnaik-Randhawa) [reply DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/JobProfile.nuspec]: Changed to 1.0.29.1

**Pull request merged**